validation-test/Sema/type_checker_perf/slow/nil_coalescing.swift.gyb

```diff
@@ -1,4 +1,4 @@
-// RUN: %scale-test --invert-result --begin 1 --end 5 --step 1 --select incrementScopeCounter %s
+// RUN: not %scale-test --invert-result --begin 1 --end 5 --step 1 --select incrementScopeCounter %s
```

Comment:
Starting returning "too complex" in different spots, didn't know how to make it work otherwise...

Reply:
Shouldn't this change instead remove `--invert-result` and move the test to `fast/`?

Reply:
Unfortunately this one didn't improve but now produces "too complex" in one of the chained "+" which I don't know how to mark with `expected-error` so I added `not` instead.

Reply:
Okay, then I think we should open a bug and just disable the test now. It's taking several minutes to run on my machine. You can feel free to merge this and then disable as a separate PR.

Reply:
Sure, I've created rdar://problem/38963783 and going to merge this all together, there is no rush.